documents/vci-flow.puml

```diff
@@ -0,0 +1,35 @@
+@startuml
```

Comment:
Stopped review on this flow diagram, as you are either mixing up things, or you are trying to merge a SIOP flow with a VCI flow, but in the demo's these are really distinct steps, meaning you should not try to do that in one flow diagram.

Given it is called the VCI flow, only stick to VCI steps.

Also mixing VCI steps with SIOP/OID4VP steps, but then only calling everything VCI frontend is very confusing to say the least

Reply:
Yeah, I was going for showing the complete thing oid4vp+oid4vci. removed the oid4vp part from this file